csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1856,22 +1859,133 @@ A program satisfies the "CVRF CSAF converter" conformance profile if:
 * For all items of `/vulernabilities[]/scores[]`: If there are CVSSv3.0 and CVSSv3.1 Vectors available for the same product, the CVRF CSAF converter shall discard the CVSSv3.0 information and provide in CSAF only the CVSSv3.1 information.
 * For all items of `/product_tree/relationships[]`: If more than one prod:FullProductName instance is given, the CVRF CSAF converter must convert the first one into the `full_product_name`. It must also output a warning that information might be lost during conversion of product relationships.
 
-## 5.7 Conformance Clause 6: CSAF post-processor
+## 5.7 Conformance Clause 6: CSAF content management system
+
+A CSAF content management system satisfies the "CSAF content management system" conformance profile if:
+
+* It satisfies the "CSAF producer" conformance profile.
+* It satisfies the "CSAF viewer" conformance profile.
+* It provides at least the following management functions:
+
+  * create new CSAF documents
+  * prefill CSAF documents based on values given in the configuration (see below)
+  * create a new version of an existing CSAF document
+  * checkout old versions of a CSAF document
+  * show diff between versions of a CSAF document
+  * list all CSAF documents within the system
+  * delete CSAF documents from the system
+  * review CSAF documents in the system
+  * approve CSAF documents
+  * search for CSAF documents by values of required fields at `document`-level or their children within the system
+  * search for CSAF documents by values of `cve` within the system
+  * search for CSAF documents based on properties of `product_tree`
+  * filter on all properties which it is required to search for
+  * export of CSAF documents
+  * show an autdit log for each CSAF document
+
+* It must identify the latest version of CSAF documents with the same `/document/tracking/id`
+* It suggests a `/document/tracking/id` based on the given configuration.
+* It keeps track of the version of CSAF documents automatically and increments for each change at least the patch_version.
```

Comment:
Suggest to replace `It keeps track` with `track`

Reply:
:+1 I change that.